libs/core/src/lib/carousel/carousel.component.html

```diff
@@ -66,14 +66,14 @@
 
 <ng-template #buttonLeft>
     <button class="fd-button fd-carousel__button fd-carousel__button--left" [style.z-index]="1" data-slide="previous"
-        [attr.aria-label]="leftNavigationBtnLabel" (click)="previous()" [disabled]="leftButtonDisabled">
+        [attr.aria-label]="leftNavigationBtnLabel" (click)="previous()" [disabled]="leftButtonDisabled" [attr.title]="leftNavigationBtnLabel">
         <i class="sap-icon--slim-arrow-left"></i>
     </button>
 </ng-template>
 
 <ng-template #buttonRight>
     <button class="fd-button fd-carousel__button fd-carousel__button--right" data-slide="next"
```

Comment:
Why you are not using `fd-button` here?

Reply:
Somehow on importing ButtonModule in Carousel Module, it breaking styles for All buttons in fundamental-ngx.
I have to investigate on this further. If it is not a big change, can we move this for now?

Reply:
@DeepakSap14 ok, but lets document it and create an issue.